index.test.js

```diff
@@ -63,11 +63,11 @@ afterAll(() => {
 
 test('run with all feature', () => {
 
-    process.env['INPUT_SERVERS'] =  '[{"id": "serverId", "username": "sUsername", "password": "sPassword", "configuration": {"props1": "value1"}}]';
-    process.env['INPUT_ORACLESERVERS'] =  '[{"id": "oServerId", "username": "oUsername", "password": "oPassword"}]';
-    process.env['INPUT_GITHUBSERVER'] =  true;
+    process.env['INPUT_SERVERS'] = '[{"id": "serverId", "username": "sUsername", "password": "sPassword", "configuration": {"props1": "value1"}}]';
```

Comment:
please add new properties to test

Reply:
I added additional tests to `settings.test.js`, and updated the `README` to mention the additional `server` properties.